src/BlockchainData.js

```diff
@@ -0,0 +1,228 @@
+import React, { Component } from 'react';
+import Web3 from 'web3';
+import Tx from 'ethereumjs-tx';
+import EthCrypto from 'eth-crypto';
+import { MULTISIG_ABI, MULTISIG_ADDRESS, MULTISIG_ERC20_ABI, MULTISIG_ERC20_ADDRESS, ERC20_ABI, 
+	TOKENIMPERIAL_ADDRESS, TOKENDEMOCRATIC_ADDRESS, TOKENIMPERIAL_SYMBOL, TOKENDEMOCRATIC_SYMBOL } from './config';
+import SmartContract from './SmartContract'; 
+import ERC20 from './erc20/ERC20';
+import './css/ButtonGroup.css';
+
+
+/* Validate forms, clear data in forms, check that transaction transfer works correctly*/ 
+class BlockchainData extends Component {
+
+	constructor(props) {
+		super(props);
+		this.state = {
+			web3: '',
+			multisig: '',
+			etherAddress: '',
```

Comment:
in your case i would store all these names in some array as constant and do all the job dynamically with map/foreach.

same for code below.